### ISSUE TITLE
Got 500 when to sign an image with v1.8.0-rc1

### ISSUE THREAD

**Issue by wy65701436** [goharbor/harbor#7779]
```
latest: digest: sha256:8fb64fee8b9f05f92c1c9fca2f3715f5973ba0c538a05463976e48a2566233c1 size: 2626
Signing and pushing trust metadata
unable to reach trust server at this time: 500.
```

**Comment by GDixon7623** [comment]
I am also getting this error

**Comment by wy65701436** [comment]
@GDixon7623 this will be resolved in the coming RC build.

**Comment by GDixon7623** [comment]
@wy65701436 Perfect thank you, do we have a rough estimate of when the build will be released?

**Comment by ninjadq** [comment]
fixed in #7785 and #7783
